app/src/main/java/org/oppia/app/testing/HtmlParserTestActivity.kt

```diff
@@ -2,14 +2,18 @@ package org.oppia.app.testing
 
 import android.os.Bundle
 import android.text.Spannable
+import android.view.View
 import android.widget.TextView
 import org.oppia.app.R
 import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.topic.conceptcard.ConceptCardFragment
 import org.oppia.util.parser.HtmlParser
 import javax.inject.Inject
 
+private const val CONCEPT_CARD_DIALOG_FRAGMENT_TAG = "CONCEPT_CARD_FRAGMENT"
+
 /** This is a dummy activity to test Html parsing. */
-class HtmlParserTestActivity : InjectableAppCompatActivity() {
+class HtmlParserTestActivity : InjectableAppCompatActivity(), HtmlParser.CustomOppiaTagActionListener {
```

Comment:
@BenHenning Can you confirm whether this testing approach is correct or not?

Reply:
This seems sensible.